### ISSUE TITLE
remove non solidity files 

### ISSUE THREAD

**Issue by ligi** [allenday/github-solidity-all#1]
there are some files in there that are no solidity files but something with solaris:

e.g.
https://github.com/allenday/github-solidity-all/blob/master/contracts/174high_droidVncServer__jni_openssl_crypto_des_times_486-50.sol

**Comment by allenday** [comment]
Yes, indeed.  I'll open an email thread with Uri, we may want to
consolidate onto his repo where he did filtering and eliminated about 50%
of the files.
